.github/workflows/e2e-every-pr.yaml

```diff
@@ -0,0 +1,31 @@
+name: Run e2e tests
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+
+env:
+  RANCHER_PASSWORD: ${{ secrets.RANCHER_PASSWORD }}
+
+jobs:
+  e2e:
+    concurrency: test-e2e
```

Comment:
This line has to be removed, we are no longer required to run only one e2e job at a time.